index.js

```diff
@@ -1,19 +1,16 @@
 const core = require('@actions/core')
-const exec = require('@actions/exec')
 const github = require('@actions/github')
 const { jirafyChangelog } = require('./utils/changelog')
 
-const src = __dirname
+var headRef = core.getInput('head-ref')
+var baseRef = core.getInput('base-ref')
+var myToken = core.getInput('myToken')
+var octokit = new github.getOctokit(myToken)
+const { owner, repo } = github.context.repo
+const regexp = /^[.A-Za-z0-9_-]*$/
```

Comment:
Perhaps even better, I ran across this: https://www.npmjs.com/package/is-git-ref-name-valid

Reply:
Git tags and commitishes could also legally include / (slash) as well as many other characters. If it's easy to handle those, we should prefer _what's allowed_ over _convention_.

The recommendation is to use just a limited set of characters (as we currently are), but at this level it may still make sense to handle more than what convention dictates, so that we don't have to scramble for a fix in the future if someone doesn't follow convention (or if convention changes).
https://docs.github.com/en/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names

Reply:
Added slash, and replaced * with +